src/phoenix/experimental/evals/models/openai.py

```diff
@@ -90,33 +118,46 @@ def _init_environment(self) -> None:
             )
 
     def _init_open_ai(self) -> None:
+        # For Azure, you need to provide the azure_endpoint
+        self._is_azure = self.azure_endpoint is not None
```

Comment:
See my other comment about `endpoint`. Also, we should exclude the empty string `""` using `bool()`.
```suggestion
        self._is_azure = bool(self.azure_deployment)
```